frontend/eda/rulebook-activations/ActivationInstanceDetails.tsx

```diff
@@ -0,0 +1,117 @@
+import { CodeBlock, CodeBlockCode, PageSection, Skeleton, Stack } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import {
+  PageDetail,
+  PageDetails,
+  PageHeader,
+  PageLayout,
+  PageTab,
+  PageTabs,
+  Scrollable,
+} from '../../../framework';
+import { formatDateString } from '../../../framework/utils/formatDateString';
+import { useGet } from '../../common/crud/useGet';
+import { RouteObj } from '../../Routes';
+import { API_PREFIX } from '../constants';
+import { EdaActivationInstance } from '../interfaces/EdaActivationInstance';
+import { EdaActivationInstanceLog } from '../interfaces/EdaActivationInstanceLog';
+import { PageDetailsSection } from '../common/PageDetailsSection';
+import React from 'react';
+import { EdaRulebookActivation } from '../interfaces/EdaRulebookActivation';
+
+export function ActivationInstanceDetails() {
+  const { t } = useTranslation();
+  const params = useParams<{ id: string }>();
+  const { data: activationInstance } = useGet<EdaActivationInstance>(
+    `${API_PREFIX}/activation-instances/${params.id ?? ''}/`
+  );
+
+  const { data: activationInstanceLog } = useGet<EdaActivationInstanceLog>(
+    `${API_PREFIX}/activation-instances/${params.id ?? ''}/logs/`
+  );
+
+  // TODO - remove when the endpoint return the activation name
```

Comment:
Is there a back end issue tracking this need?  If so, can we make an issue for the UI side and link the 2 issues so that we don't loose track of the tasks needed?  Then, can we remove this comment?

Reply:
@AlexSCorey - yes, there is a Jira story for the backend, I will remove the comment